src/ngResource/resource.js

```diff
@@ -349,6 +367,52 @@ angular.module('ngResource', ['ng']).
   provider('$resource', function() {
     var provider = this;
 
+    /**
+     * @ngdoc property
+     * @name $resourceProvider#defaults
+     * @description
+     * A configuration object for `$resource` instances.
+     *
+     * Properties of this object are initialized with a set of values that satisfies a wide range of use cases.
+     * User can also choose to override these properties in application configuration phase.
+     *
+     * Property `stripTrailingSlashes`, default to true, strips trailing slashes from
+     * calculated URLs.
+     *
+     * Property `actions` is an object that sets up high level methods/aliases on `$resource` object
+     * based on standard HTTP methods. Users can supply an "actions" object with method aliases that
+     * comply with alternative conventions.
+     *
+     * To add your own set of configurations, set it up like so:
+     * ```
+     * angular.module('myApp').config(['resourceProvider', function ($resourceProvider){
+     *
+     *    // Provide your own set of actions on $resource factory.
+     *    // The following comments are Angular's default actions, which are being
+     *    // replaced by an object that includes a PUT method as shown.
+     *    //  { 'get':    {method:'GET'},
+     *    //	'save':   {method:'POST'},
+     *    //	'query':  {method:'GET', isArray:true},
+     *    //	'remove': {method:'DELETE'},
+     *    //	'delete': {method:'DELETE'} };
+     *
+     *    $resourceProvider.defaults.actions = {
+     *        create:{method: 'POST'},
+     *        save:	 {method: 'POST'},
+     *        update:{method: 'PUT'},
+     *        get:	 {method: 'GET'},
+     *        query:    {method: 'GET', isArray:true},
+     *        remove:   {method: 'DELETE'},
+     *        delete:   {method: 'DELETE'}
+     *    };
+     *
+     *    // Don't strip trailing slashes from calculated URLs.
+     *    // Consult your application server's configuration to work in concert with this setting.
+     *    $resourceProvider.defaults.stripTrailingSlashes = false;
+     * }]);
+     * ```
+     *
+     */
```

Comment:
Could you bring this more inline with how [`$cookiesProvider.defaults`][1] are formatted ?

[1]: https://github.com/angular/angular.js/blob/v1.4.0-rc.2/src/ngCookies/cookies.js#L28-L49